<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAreaWidget</class>
 <widget class="QWidget" name="GetAreaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Area</string>
   </property>
  </widget>
  <widget class="QLabel" name="areaLabelImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>img</string>
   </property>
   <property name="locale">
    <locale language="German" country="Germany"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="areaHorizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>50</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string extracomment="sss"/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="sliderPosition">
    <number>30</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
